Gemfile.lock

```diff
@@ -1,77 +1,107 @@
 PATH
   remote: .
   specs:
-    lr_common_styles (2.0.0)
-      bootstrap-sass (~> 3.4.0)
-      font-awesome-rails (~> 4.7.0.1)
-      govuk_elements_rails (~> 2.0.0)
-      govuk_frontend_toolkit (~> 4.18.1)
-      govuk_template (~> 0.18.1)
-      haml-rails (~> 2.0.0)
-      jquery-rails (>= 4.3.5, < 4.7.0)
-      lodash-rails (~> 4.17.14)
-      modernizr-rails (~> 2.7.1)
-      modulejs-rails (~> 2.2.0.0)
-      rails (>= 5.2.4)
-      sass-rails (~> 5.0.4)
+    lr_common_styles (2.1.0)
+      bootstrap-sass
```

Comment:
What is the reason for removing any version specification?

Reply:
Jon had the same question on the Qonsole PR, I replied to him here: https://github.com/epimorphics/qonsole-rails/pull/53#discussion_r1812985477

Reply:
Okay. I understand and it makes sense for the pure code packages. I initially thought about it because something like bootstrap can have big changes between versions and we could wind up with broken CSS

Reply:
Good point about the CSS. Just noticed you reviewed the `Gemfile.lock` as well. There is still a version specification in there on line `112` (`bootstrap-sass (3.4.1)`). I removed the version specification from the gemspec, so unless someone nukes `Gemfile.lock` or runs `bundle update bootstrap-sass` `bootstrap-sass` will never update